core/cmd/client.go

```diff
@@ -265,7 +265,8 @@ func (t *SessionCookieAuthenticator) Authenticate(sessionRequest models.SessionR
 	if len(cookies) == 0 {
 		return nil, errors.New("Did not receive cookie with session id")
 	}
-	return cookies[0], t.store.Save(cookies[0])
+	sc := web.FindSession(cookies)
+	return sc, t.store.Save(sc)
```

Comment:
What do you think about improving the name of web.FindSession(..)?  I cant set up a small PR for this after this is merged